src/styles/_catalog.scss

```
@@ -1,22 +1,37 @@
 .catalog {
-  display: flex;
-  flex-wrap: wrap;
+  display: grid;
+  grid-template-columns: repeat(1, minmax(#{$card-width}, 1fr));
+  gap: 48px;
 
   max-width: 944px;
   margin: 0 auto;
 
   &__card {
-    display: flex;
+    display: grid;
     justify-content: center;
     align-items: center;
-    width: 200px;
+    width: #{$card-width};
     height: 300px;
     border: 1px solid #ccc;
     border-radius: 5px;
     margin-bottom: 48px;
   }
+}
+
+@media (min-width: 488px) and (max-width: 767px) {
+  .catalog {
+    grid-template-columns: repeat(2, minmax(#{$card-width}, 1fr));
+  }
+}
```

```suggestion
.catalog {
  @media (min-width: 488px) and (max-width: 767px) {
    grid-template-columns: repeat(2, $card-width);
  }
}
```
move this inside upper catalog